src/main/java/net/dv8tion/jda/core/entities/Emote.java

```diff
@@ -135,6 +135,15 @@
      */
     EmoteManagerUpdatable getManagerUpdatable();
 
+    /**
+     * Whether or not this Emote is animated or not.
```

Comment:
Redundant

Reply:
Actually its a good idea I actually use the emoji part and think adding support for emoji animation detection is needed

Reply:
`Whether this emote is animated.`